### PR TITLE
Bump dependency versions

### PR DESCRIPTION
**Pull request by aalmiray**
Related to #38

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,10 +25,11 @@
...
 		<junit-jupiter.version>5.8.2</junit-jupiter.version>
 		<assertj-core.version>3.22.0</assertj-core.version>
-		<jackson.version>2.15.0</jackson.version>
-		<slf4j.version>2.0.7</slf4j.version>
-		<json.schema.version>1.0.80</json.schema.version>
+		<jackson.version>2.16.1</jackson.version>
+		<slf4j.version>2.0.9</slf4j.version>
+		<json.schema.version>1.3.2</json.schema.version>
 		<packageurl.version>1.4.1</packageurl.version>
+		<commons-lang3.version>3.14.0</commons-lang3.version>
```
Comment (afrittoli) [inline pom.xml:32]: π version ^_^

File: generator/pom.xml [hunk 2]
```diff
@@ -46,7 +46,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <version>3.12.0</version>
+            <version>${commons-lang3.version}</version>
```
Comment (afrittoli) [inline generator/pom.xml:49]: Does this mean that the version used by the generator and the SDK must be the same now?
I don't mind, we can still split it future if needed, right?

**Review by afrittoli — APPROVED** [review]
Thanks. @rjalander any comment on your side?

File: generator/pom.xml (hunk 2 above)
Comment (aalmiray) [reply generator/pom.xml:49]: They don't have to match per se but it makes sense to keep it this way if there were to be any CVEs in one version fixed in another.

**Review by rjalander — APPROVED** [review]

**Pull request merged**